src/sage/groups/braid.py

```diff
@@ -1517,8 +1539,10 @@ def _left_normal_form_coxeter(self):
         return tuple([-delta] + form)
 
     def right_normal_form(self):
-        """
-        Return the right normal form of the braid.
+        r"""
+        A tuple of simple generators in the right normal form. The last
```

Comment:
You could leave the original "Return the right normal form" and have your additions on the next line after a blank line.